src/bp/core/services/module/resources-loader.ts

```diff
@@ -28,6 +38,13 @@ export class ModuleResourceLoader {
 
   constructor(private logger: Logger, private moduleName: string, private ghost: GhostService) {}
 
+  async runMigrations() {
+    const mfile = `${this.modulePath}/migrations.json`
+    if (fse.existsSync(mfile)) {
+      await this._executeMigration(mfile)
+    }
+  }
```

Comment:
Looks like it will execute the migrations everytime the server starts. Is that expected ?

Reply:
for now yes, until we hold a more official migration registry / history